src/core/client/admin/test/configure/moderation.spec.tsx

```diff
@@ -203,14 +203,6 @@ it("change akismet settings", async () => {
   // Input valid api key
   keyField.props.onChange("my api key");
 
-  // Input malformed site.
-  siteField.props.onChange("malformed url");
-
-  expect(
-    within(akismetContainer).queryAllByText("This field is required.").length
-  ).toBe(0);
-  expect(within(akismetContainer).queryAllByText("Invalid URL").length).toBe(1);
```

Comment:
If none of the form's validation fields changed, why are these expectations removed?

Reply:
Validation logic was changed:

https://github.com/coralproject/talk/pull/2425/files#diff-194a472865e421b58154dea9572ee27dL92